json-schema-validator-all/2.2.6.wso2v1/pom.xml

```diff
@@ -0,0 +1,142 @@
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            com.github.fge;version="${version.btf}",
+                            com.github.fge.jackson.*;version="${version.jackson.coreutils}",
+                            com.github.fge.jsonschema.core.*;version="${version.json.schema.core}",
+                            com.github.fge.jsonschema.*;version="${version.json.schema.validator}",
+                            com.github.fge.msgsimple.*;version="${version.msg.simple}",
+                            com.github.fge.uritemplate.*;version="${version.uri.template}",
+                        </Export-Package>
+                        <Import-Package>
+                            !com.github.fge.*,
+                            com.google.common.base; version="${version.com.google.common}",
+                            com.fasterxml.jackson.core; version="${version.com.fasterxml.jackson.core}",
+                            com.google.common.collect; version="${version.com.google.common}",
+                            com.fasterxml.jackson.databind; version="${version.jackson.databind}",
+                            com.fasterxml.jackson.databind.node; version="${version.jackson.databind}",
+                            com.google.common.io; version="${version.com.google.common}",
+                            org.joda.time.format; version="${version.org.joda.time}",
+                            org.joda.time; version="${version.org.joda.time}",
+                            javax.mail.internet; version="${version.javax.mail.internet}",
+                            com.google.common.cache; version="${version.com.google.common}",
+                            com.google.common.primitives; version="${version.com.google.common}",
+                            com.google.i18n.phonenumbers; version="${version.com.google.i18n.phonenumber}",
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            @json-schema-core-${version.json.schema.core}.jar!/draftv4/*,
+                            @json-schema-core-${version.json.schema.core}.jar!/draftv3/*
+                        </Include-Resource>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <version.json.schema.validator>2.2.6</version.json.schema.validator>
+        <version.json.schema.core>1.2.5</version.json.schema.core>
+        <version.uri.template>0.9</version.uri.template>
+        <version.msg.simple>1.1</version.msg.simple>
+        <version.btf>1.2</version.btf>
+        <version.jackson.coreutils>1.8</version.jackson.coreutils>
```

Comment:
As a general practice, the version is mentioned at the end of the property. Shall we follow the same approach as https://github.com/wso2/orbit/pull/190/files